ui/window.go

```diff
@@ -115,6 +115,11 @@ func (win *Window) changePage(page string) {
 	win.window.Invalidate()
 }
 
+func (win *Window) pushNavigationPage(from string) {
```

Comment:
"pushNavigationPage" is a bit misleading as push and pop are words associated with arrays. In this case you're simply setting a value. It can be "setReturnPage"